src/data_loader.py

```diff
@@ -97,6 +118,12 @@ def delete_dataset(self, schema_id):
 
         # Clean up the access & dataset tables
         try:
+            id = schema_id[-1:]
+            query = cursor.mogrify('INSERT INTO Available_Schema (id) VALUES (%s)', id)
```

Comment:
Since `id` is an integer in the table, you should convert `id` to an integer (either when inserting it or when creating it), otherwise it will be inserted as a string. I'm not sure if this is a problem (will Postgres complain about it?) but it's better to be safe than sorry, imo.

Reply:
Converting to int gives TypeError. Mogrify makes sure there is no issue.